cmd/argocd/commands/admin/app.go

```diff
@@ -295,6 +296,7 @@ func NewReconcileCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command
 	command.Flags().StringVar(&selector, "l", "", "Label selector")
 	command.Flags().StringVar(&outputFormat, "o", "yaml", "Output format (yaml|json)")
 	command.Flags().BoolVar(&refresh, "refresh", false, "If set to true then recalculates apps reconciliation")
+	command.Flags().BoolVar(&serverSideDiff, "serverSideDiff", false, "If set to true will use serverSideDiff while comparing resources")
```

Comment:
Maybe `server-side-diff` to follow the pattern of the other flag?

Reply:
I am not sure as this isn't necessarily a "feature flag" but a normal CLI flag. The difference is that this command is executed by humans/automation while the controller command is just executed as a container.
What is better in this case?
- `argocd admin app get-reconcile-results --server-side-diff-enabled=true`
- `argocd admin app get-reconcile-results --server-side-diff=true``
WDYT?

Reply:
done